package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "./src/server.js",
   "scripts": {
-    "start": "nodemon -e ts --exec \"npm run compile\"",
-    "compile": "cti create -i BaseEntity.ts ./src&& (tsc || exit 1) && node ./build/server.js"
+    "start": "npm run compile && nodemon ./build/server.js",
+    "compile": "tsc"
```

Comment:
Yea, I had these scripts poorly named but did not get around to fixing them, I like have nodemon running on the compile though since it reported the errors save and auto restarted.

Reply:
Oh, so you could do something like `"compile": "nodemon tsc"`? I am not that familiar with `nodemon` so I am not sure how to structure that script.